### ISSUE TITLE
TypeError when trying to edit Knowledge Base plugin article

### ISSUE THREAD

**Issue by PacifistRiot** [codefoxes/greenlet#42]
Using WP 5.5.1, [echo knowledge base plugin](https://wordpress.org/plugins/echo-knowledge-base/), and Greenlet 2.0.1.

Trying to edit a knowledge base article leads to the following error preventing editing.
```
TypeError: Cannot use 'in' operator to search for 'template' in undefined
    at https://website.com/wp-content/themes/greenlet/library/backend/assets/js/greenlet-meta.min.js?ver=2.0.1:1:1432
    at je (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:78:476)
    at ph (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:215:173)
    at lh (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:126:409)
    at O (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:121:71)
    at ze (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:118:14)
    at https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:53:49
    at unstable_runWithPriority (https://website.com/wp-includes/js/dist/vendor/react.min.js?ver=16.9.0:26:340)
    at Ma (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:280)
    at mg (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:496)

TypeError: Cannot use 'in' operator to search for 'template' in undefined
    at https://website.com/wp-content/themes/greenlet/library/backend/assets/js/greenlet-meta.min.js?ver=2.0.1:1:1432
    at je (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:78:476)
    at ph (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:215:173)
    at lh (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:126:409)
    at O (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:121:71)
    at ze (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:118:14)
    at https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:53:49
    at unstable_runWithPriority (https://website.com/wp-includes/js/dist/vendor/react.min.js?ver=16.9.0:26:340)
    at Ma (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:280)
    at mg (https://website.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:496)
```

**Comment by karthikax** [comment]
Thanks for reporting the issue @PacifistRiot 
Able to reproduce the issue. We should have a fix soon

**Comment by karthikax** [comment]
Attaching a temporary fix here
Download these 2 files to https://website.com/wp-content/themes/greenlet/library/backend/assets/js/
[greenlet-meta.js](https://gist.githubusercontent.com/karthikax/13e6c13f9aa54b1f4637eb5a352bdec1/raw/36d98e2c869f45576f60b70150155c5282e1a2fa/greenlet-meta.js)
[greenlet-meta.min.js](https://gist.githubusercontent.com/karthikax/13e6c13f9aa54b1f4637eb5a352bdec1/raw/36d98e2c869f45576f60b70150155c5282e1a2fa/greenlet-meta.min.js)
should fix the issue temporarily.

Adding this to our backlog for a permanent fix

**Comment by karthikax** [comment]
This issue has been fixed with version 2.1.0
